ParquetClassLibrary/Items/AllItems.cs

```diff
@@ -0,0 +1,106 @@
+using System;
+using System.Collections.Generic;
+
+namespace ParquetClassLibrary.Items
+{
+    /// <summary>
+    /// Stores all defined <see cref="Item"/>s.
+    /// This collection is the source of truth about items for the rest of the library,
+    /// something like a color palette that other classes can paint with.
+    /// </summary>
+    public static class AllItems
```

Comment:
nit: It feels as though the contents of this class is entirely a wrapper for EntityCollections, providing a static interface. I wonder if this class could exist as a static property of a common library location, rather than getting its entire own class.

If more functions and utility is added to this class in the future, it has definitely earned its own place.

Reply:
Hmmmmmmm, that is a very good point!
I will think about that!  #66